vite.config.ts

```diff
@@ -1,10 +1,24 @@
 import { defineConfig } from "vitest/config";
 import { sveltekit } from "@sveltejs/kit/vite";
+import { svelteTesting } from "@testing-library/svelte/vite";
 
 export default defineConfig({
-    plugins: [sveltekit()],
-
+    plugins: [sveltekit(), svelteTesting()],
+    // Tell Vitest to use the `browser` entry points in `package.json` files, even though it's running in Node
+    resolve: process.env.VITEST ? { conditions: ["browser"] } : undefined,
     test: {
-        include: ["test/**/*.{test,spec}.{js,ts}"],
+        include: ["test/**/*.{test,spec}.{js,ts}", "integration/**/*.{test,spec}.{js,ts}"],
```

Comment:
Do we have an integration directory? For components testing?

Reply:
Currently, we don't because we have no tests but as soon as we add them, e.g. in #74 this directory will also be tracked in git.